ui-tests/cypress/support/restAPI/abstractController.js

```diff
@@ -5,8 +5,8 @@ export default class AbstractController {
     this.accessToken = accessToken;
   }
 
-  intercept(routeMatcher, alias) {
-    routeMatcher.url = this.apiURL;
+  intercept(routeMatcher, path, alias) {
+    routeMatcher.url = this.apiURL+path;
     return cy.intercept(routeMatcher).as(alias);
```

Comment:
At first, I tried to pass a routeMatcher object with a 'path' property, however when using a 'path' property, it's expected to also have a 'hostname' property. Unfortunately, the apiURL can't be used as 'hostname', because 1) it has "http://" at the start of the string, and 2) it's not just the hostname. Therefore, I decided to use this slightly roundabout way, so that the 'url' property is made from using the apiURL and the 'path' passed as argument.